tsl/test/sql/CMakeLists.txt

```diff
@@ -106,8 +106,6 @@ endif(CMAKE_BUILD_TYPE MATCHES Debug)
 if((${PG_VERSION_MAJOR} GREATER_EQUAL "14"))
   if(CMAKE_BUILD_TYPE MATCHES Debug)
     list(APPEND TEST_FILES chunk_utils_internal.sql)
-    list(APPEND TEST_TEMPLATES continuous_aggs.sql.in
-         continuous_aggs_deprecated.sql)
```

Comment:
You have added the new input file `continuous_aggs_deprecated.sql.in`, but I do not see the old file `continuous_aggs_deprecated.sql` removed.

Reply:
Thanks, now removed.